stl/inc/algorithm

```diff
@@ -4072,9 +4072,12 @@ namespace ranges {
 
     // VARIABLE ranges::unique_copy
     // clang-format off
+    template <class _It, class _Ty>
+    concept _Is_input_with_value_type = input_iterator<_It> && same_as<iter_value_t<_It>, _Ty>;

```

Comment:
I am wondering whether this should be `_Is_input_with_matching_value_type `

Reply:
I'm not married to this name. There are only two uses and they're both within 50 lines of this definition, so it doesn't need to be perfect.